### DISCUSSION TITLE
Exposing API schema

### DISCUSSION THREAD

**Discussion by isBatak** [kurierjs/kurier#267] [Ideas]
How hard would it be to expose resource schemas via API so the frontend can build a `CLI` tool for building Resource definitions used in frontend clients?

**Comment by joelalejandro** [comment]
It shouldn't be a problem. We could add something like a `_schema` endpoint, following on the initial conventions defined in json-api/json-api#1447.

**Reply by isBatak** [reply]
This is cool... I saw `describedby` in 1.1 documentation but haven't realized it's for exposing schema definitions.
I'll have it in mind.
